sdk/src/Core/AWSSDK.Core.NetFramework.csproj

```diff
@@ -69,6 +69,6 @@
   <ItemGroup>
     <PackageReference Include="System.Buffers" Version="4.5.1" />
     <PackageReference Include="System.Memory" Version="4.5.5" />
-    <PackageReference Include="System.Text.Json" Version="8.0.3" />
+    <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
I bumped up this package version because 8.0.3 was being flagged for a vulnerability. Since we have enabled "treat warning as errors", this led to compilation failures